src/NuProj.Targets/NuProj.targets

```diff
@@ -148,7 +150,7 @@
     <ProjectReference>
       <PackageDirectory>Lib</PackageDirectory>
       <TargetFramework></TargetFramework>
-      <Targets>$(ProjectOutputGroups)</Targets>
+      <ProjectOutputGroups>$(ProjectOutputGroups)</ProjectOutputGroups>
```

Comment:
MSBuild task https://github.com/nuproj/nuproj/pull/193/files#diff-d02cfc639708fa90c22ed5a0d579033eL273 is duplicate of `c:\Program Files (x86)\MSBuild\14.0\Bin\Microsoft.Common.CurrentVersion.targets`. So I removed it and kept the common one, which uses `Targets="%(_MSBuildProjectReferenceExistent.Targets)"`. Target containing this task is used to build references (when not building in VS).